nilearn/decoding/searchlight.py

```diff
@@ -176,7 +177,7 @@ def _group_iter_search_light(list_rows, estimator, X, y,
                 else:
                     crlf = "\n"
                 percent = float(i) / len(list_rows)
-                percent = round(percent * 100, 2)
+                percent = np.round(percent * 100, 2)
```

Comment:
My understanding is that the round -> np.round replacement is only needed when the thing you are rounding is a numpy scalar. It doesn't seem necessary here.